src/helm/benchmark/slurm_runner.py

```diff
@@ -0,0 +1,220 @@
+import dataclasses
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Dict, List
+import argparse
+import json
+import os
+import time
+import sys
+
+import dacite
+
+from helm.benchmark.executor import ExecutionSpec
+from helm.benchmark.runner import Runner, RunSpec, RunnerError
+from helm.benchmark.slurm_jobs import (
+    submit_slurm_job,
+    cancel_slurm_job,
+    get_slurm_job_state,
+    SlurmJobState,
+    TERMINAL_SLURM_JOB_STATES,
+    FAILURE_SLURM_JOB_STATES,
+)
+from helm.common.general import ensure_directory_exists, asdict_without_nones
+from helm.common.hierarchical_logger import hlog
+
+
+@dataclass
+class SlurmRunnerSpec:
+    """Arguments to instantiate a SlurmRunner."""
+
+    execution_spec: ExecutionSpec
+    output_path: str
+    suite: str
+    skip_instances: bool
+    cache_instances: bool
+    cache_instances_only: bool
```

Comment:
What's the difference?

Reply:
Hm, having both flags looks pretty confusing...can flag as something that can be cleaned up later.